### PR TITLE
gymclass tests

### PR DESCRIPTION
**Pull request by sameerkhoja**

### CODE REVIEW

File: src/app/dao/gymclassinstance_dao.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def get_gym_class_instance_by_id(gym_class_instance_id):
 
 def get_gym_class_instances_by_gym_class(gym_class_name, page, page_size=10):
   return GymClassInstance.query.filter(
-      GymClassInstance.gym_class == gym_class_name
+      GymClassInstance.gym_class_id == gym_class_name
```
Comment (young-k) [inline src/app/dao/gymclassinstance_dao.py]: Hm, this is inconsistent –– you're comparing the gym_class_name to gym_class_id. Maybe change the instance var name to be `gym_class_id` as well?

File: tests/test_gymclasses.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import datetime as dt
```
Comment (young-k) [inline tests/test_gymclasses.py]: Do you use `datetime` anywhere here?

File: tests/test_gymclasses.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+from flask import json
```
Comment (young-k) [inline tests/test_gymclasses.py]: Same with `json`

File: tests/test_gymclassinstances.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    }
+    test_cd = cdd.create_class_desc('test_class', 'test_class_desc')[1]
+    test_cd2 = cdd.create_class_desc('test_class2', 'test_class_desc2')[1]
+    test_gymclassinstance = gcid.create_gym_class_instance(test_args)[1]
+    test_gymclassinstance2 = gcid.create_gym_class_instance(test_args2)[1]
+    self.assertEquals(test_gymclassinstance, gcid.get_gym_class_instance_by_id(test_gymclassinstance.id))
+    self.assertEquals(test_gymclassinstance2, gcid.get_gym_class_instance_by_id(test_gymclassinstance2.id))
+    self.assertIn(test_gymclassinstance, gcid.get_all_gym_class_instances(None))
+    self.assertIn(test_gymclassinstance2, gcid.get_all_gym_class_instances(None))
+    self.assertIn(test_gymclassinstance, gcid.get_gym_class_by_start_duration(test_gymclassinstance.gym_class_id, test_gymclassinstance.gym_id, test_gymclassinstance.start_dt, test_gymclassinstance.duration))
```
Comment (young-k) [inline tests/test_gymclassinstances.py]: These lines are way too long, cap everything at 80 columns

File: tests/test_gymclassinstances.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import datetime as dt
```
Comment (young-k) [inline tests/test_gymclassinstances.py]: Again, same thing with these imports

File: tests/test_gymclassinstances.py [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+from flask import json
+from tests.test_case import *
+from app.dao import gymclassinstance_dao as gcid
+from app.dao import gymclass_dao as gcd
+from app.dao import instructors_dao as insd
```
Comment (young-k) [inline tests/test_gymclassinstances.py]: Cursory glance, but doesn't seem like you use this import either

Comment (sameerkhoja) [reply tests/test_gymclassinstances.py]: Fixed changes!

File: tests/test_gymclassinstances.py [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+        "instructor_name": "test_instructor2",
+        "is_cancelled": True,
+        "start_time": "4:00PM",
+        "end_time": "5:00PM",
+    }
+    test_cd = cdd.create_class_desc('test_class', 'test_class_desc')[1]
+    test_cd2 = cdd.create_class_desc('test_class2', 'test_class_desc2')[1]
+    test_gymclassinstance = gcid.create_gym_class_instance(test_args)[1]
+    test_gymclassinstance2 = gcid.create_gym_class_instance(test_args2)[1]
+    self.assertEquals(test_gymclassinstance,
```
Comment (young-k) [inline tests/test_gymclassinstances.py]: Can you make this:
```
self.assertEquals(
    test_gymclassinstance,
...
)
```
So put the closing parens on a line by itself, and put the first one on its own line also –– do this for everything under this too!

**Pull request merged**